lib/features/user/view_model/user_view_model.dart

```diff
@@ -81,61 +91,72 @@ class UserViewModel extends _$UserViewModel {
     if (USE_MOCK_DATA) {
       // Simulate a successful update with mock data
       final updatedUser = userMock.copyWith(bio: newBio);
-      ref.read(authLocalRepositoryProvider).setUser(updatedUser); // Persist the change
+      ref
+          .read(authLocalRepositoryProvider)
+          .setUser(updatedUser); // Persist the change
       ref.read(userProvider.notifier).update((_) => updatedUser);
       state = UserState.success('Bio updated successfully');
       return;
     }
 
-    final response = await ref.read(userRemoteRepositoryProvider).updateBio(newBio: newBio);
+    final response =
+        await ref.read(userRemoteRepositoryProvider).updateBio(newBio: newBio);
 
     response.fold(
-          (appError) {
+      (appError) {
         state = UserState.fail(appError.error);
       },
-          (_) {
+      (_) {
```

Comment:
as you are using this brackets (_) in this way, then you are returning Either<AppError, void>
don't do that, just return 'AppError?', and check if it is null or not